src/Administration/Resources/app/administration/src/core/factory/http.factory.js

```diff
@@ -67,15 +67,15 @@ function createClient() {
  * @returns {AxiosInstance}
  */
```

Comment:
@jleifeld the cache adapter is unfortunately different in the new axios, would be great if you could take a look into that